demos/serverless/src/handlers.js

```diff
@@ -127,7 +127,12 @@ exports.logs = async (event, context) => {
     });
   }
   putLogEventsInput.logEvents = logEvents;
-  await cloudWatchClient.putLogEvents(putLogEventsInput).promise();
+  try {
+    await cloudWatchClient.putLogEvents(putLogEventsInput).promise();
+  } catch (error) {
+    console.error(error.message + ' - Failed to write to cloudwatch using token ' + uploadSequence + ' and the LogEvent:');
```

Comment:
How about logging in one line?
```
console.error(`Failed to put CloudWatch log events with error ${error.message} and params: ', putLogEventsInput);

// or stringify params if Lambda doesn't print the 2nd parameter well.

console.error(`Failed to put CloudWatch log events with error ${error.message} and params ${JSON.stringify(putLogEventsInput)});
```

Incorporating:
- String templates ``
- `putLogEventsInput` already includes `uploadSequence`
- cloudwatch => CloudWatch